xbmc/filesystem/AudioBookFileDirectory.cpp

```diff
@@ -75,6 +75,53 @@ bool CAudioBookFileDirectory::GetDirectory(const CURL& url,
   if (m_fctx->nb_chapters > 1)
     thumb = IMAGE_FILES::URLFromFile(url.Get(), "music");
 
+  //Look for any embedded art
```

Comment:
Blank between `//` and `Look` => https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#9-comments

Reply:
Done